includes/wikia/services/ArticleQualityService.php

```diff
@@ -158,6 +159,18 @@ public function getArticleQuality() {
 			$article = new Article( $title );
 			$parser = $article->getParserOutput();
 
+			if ( !$parser ) {
+				//MAIN-3592
+				WikiaLogger::instance()->error(
+					__METHOD__,
+					[
+						'message' => 'Article::getParserOutput returned false',
+						'articleId' => $this->articleId,
+					]
+				);
+				return null;
```

Comment:
how this change will influance indexer output, and therefore article quality field in solr index?